scripts/bash/run_standardization_conformance.sh

```diff
@@ -17,6 +17,7 @@ SRC_DIR=$(dirname "$0")
 
 source ${SRC_DIR}/enceladus_const.sh
 source ${SRC_DIR}/enceladus_env.sh
+source ${SRC_DIR}/version_file.sh
```

Comment:
Still calling the file without underscore.

Reply:
I always forget about the joint script. I still did not find a place in my heart for it.

Reply:
Fixed

Reply:
> 
> 
> I always forget about the joint script. I still did not find a place in my heart for it.

Th youngest one is always the least love in the fairy tales. Especially by a foster-parent. 😆